components/core/tests/test-sql.cpp

```diff
@@ -0,0 +1,23 @@
+#include <sstream>
+
+#include <Catch2/single_include/catch2/catch.hpp>
+#include <spdlog/spdlog.h>
+
+#include "../src/clp_s/search/EmptyExpr.hpp"
+#include "../src/clp_s/search/sql/sql.hpp"
+#include "LogSuppressor.hpp"
+
+using clp_s::search::EmptyExpr;
+using clp_s::search::sql::parse_sql_expression;
+using std::stringstream;
+
+TEST_CASE("Test parsing SQL", "[SQL]") {
+    // Suppress logging
+    LogSuppressor suppressor{};
```

Comment:
Not used?

Reply:
It's use is entirely implemented in the constructor/destructor. It just turns off spdlog logging once created, and re-enables it once destroyed.

Just allows you to run parsing unit tests without emitting tons of log messages for parsing failure.